src/dmd/marsparams.d

```diff
@@ -0,0 +1,30 @@
+/**
+ * This modules defines the compiler parameters that are specific to DMD.
+ * This file is not shared with other compilers which use the DMD front-end.
+ *
+ * Copyright:   Copyright (C) 1999-2020 by The D Language Foundation, All Rights Reserved
+ * Authors:     $(LINK2 http://www.digitalmars.com, Walter Bright)
+ * License:     $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Source:      $(LINK2 https://github.com/dlang/dmd/blob/master/src/dmd/marsparams.d, _marsparams.d)
+ * Documentation:  https://dlang.org/phobos/dmd_marsparams.html
+ * Coverage:    https://codecov.io/gh/dlang/dmd/src/master/src/dmd/marsparams.d
+ */
+
+module dmd.marsparams;
```

Comment:
What is the point of this module if it is unused?  Perhaps you meant to remove the struct definition from `dmd.mars`?

Reply:
I... I forgot the unstage it hahaha.